torchtext/experimental/datasets/sequence_tagging.py

```diff
@@ -0,0 +1,220 @@
+import torch
+
+from torchtext.experimental.datasets import raw
+from torchtext.vocab import build_vocab_from_iterator
+from torchtext.experimental.functional import (
+    vocab_func,
+    totensor,
+    sequential_transforms,
+)
+
+
+def _build_vocab(data, word_transform):
+    total_columns = len(data[0])
+    data_list = [[] for _ in range(total_columns)]
+    vocabs = []
+
+    for line in data:
+        for idx, col in enumerate(line):
+            if idx == 0:
+                col = word_transform(col) if word_transform else col
+                data_list[idx].append(col)
+            else:
+                data_list[idx].append(col)
+
+    for it in data_list:
+        vocabs.append(build_vocab_from_iterator(it))
+
+    return vocabs
+
+
+def _setup_datasets(dataset_name,
+                    train_filename,
+                    valid_filename,
+                    test_filename,
+                    separator,
+                    data_select=("train", "valid", "test"),
+                    root=".data",
+                    vocabs=None,
+                    word_tokenizer=None):
+    train, val, test = DATASETS[dataset_name](train_filename=train_filename,
+                                              valid_filename=valid_filename,
+                                              test_filename=test_filename,
+                                              root=root)
+    raw_data = {
+        "train": [line for line in train] if train else None,
+        "valid": [line for line in val] if val else None,
+        "test": [line for line in test] if test else None
+    }
+
+    word_transform = None
+    if word_tokenizer:
```

Comment:
What happen if `tokenizer` is None?

Reply:
this is the sample of the data
```
Al	PROPN	NNP
-	PUNCT	HYPH
Zaman	PROPN	NNP
:	PUNCT	:
American	ADJ	JJ
forces	NOUN	NNS
killed	VERB	VBD
Shaikh	PROPN	NNP
Abdullah	PROPN	NNP
al	PROPN	NNP
-	PUNCT	HYPH
```

we don't need a tokenizer as per usual as the words themselves are already tokenized since the beginning, so I believe setting up a default tokenizer wouldn't make sense in this case

Reply:
If we don't need a tokenizer for this dataset, should we just delete them?

Reply:
I was thinking to keep them so that it allows user to try different tokenizer for the word, such as adding capability to use char level embedding instead of word. Or do you have other way to achieve similar results?

Reply:
After some testing it seems to use char level embedding is a bit hard, might be better to use transform function instead. Still not pretty sure so I just remove the tokenizer for now